spec/views/forms/index.html.erb_spec.rb

```diff
@@ -14,7 +16,7 @@
     end
 
     it "does not contain a a list of forms" do
-      expect(rendered).not_to have_text "Your forms"
+      expect(rendered).not_to have_table
       expect(rendered).not_to have_css ".govuk-summary-list"
```

Comment:
Is this check still needed? I thought we only using a table on the page now.

Reply:
I don't know, if that was the case that change was made before I joined the team...

Reply:
I suspect this is overhang when the list of forms were using summary list component. It might be worth removing this line as we aren't using summary list component any more. 

This is one reason why we need to be careful using `not_to` in the specs they can result in false positives.

Reply:
I've removed it.